docs/resources/Channel.md

```diff
@@ -337,6 +340,16 @@ Represents a message sent in a channel within Discord.
 | EPHEMERAL              | 1 << 6 | this message is only visible to the user who invoked the Interaction              |
 | LOADING                | 1 << 7 | this message is an Interaction Response and the bot is "thinking"                 |
 
+###### Message `sticker_items` Structure
```

Comment:
```suggestion
###### Message Sticker Item Structure
```
this might make more sense + I don't think ` is used in any headers

Reply:
Yeah I adjusted, ty for noticing!